terraform/api_gateway.tf.json.template.py

```diff
@@ -174,7 +174,8 @@ def for_domain(cls, domain):
                                     ]))
                                 }
                             },
-                            "tags": get_aws_tags(f'cloudwatch_{lambda_.name}_group'),
+                            "tags": get_cloud_tags(
+                                name="/aws/apigateway" + config.qualified_resource_name(lambda_.name)),
```

Comment:
I'm still curious about these special cases. What's the formula or convention for naming tags explicitly?

Reply:
I'm following these guidelines:
* If the resource already has a name assigned to it, use that value. This value should already be qualified.
* If the resource does not already have a name assigned to it, create a unique value in the qualified format (e.g., azul-foo-dev)
* Each name should be unique. (Unless this would deviate from the original resource name)

I've noted general guidelines [here](https://github.com/DataBiosphere/azul/issues/1552#issuecomment-616796430). Should I also copy those guidelines into this PR?

Reply:
Yeah I think it would be useful to add this to the docstring of the `get_cloud_tags()` function.